FreeRTOS/Demo/Posix_GCC/main.c

```diff
@@ -140,11 +140,11 @@ static void handle_sigint( int signal );
 StackType_t uxTimerTaskStack[ configTIMER_TASK_STACK_DEPTH ];
 
 /* Notes if the trace is running or not. */
-#if ( projCOVERAGE_TEST == 1 )
+#if ( projENABLE_TRACING == 0 )
     static BaseType_t xTraceRunning = pdFALSE;
-#else
+#else /* if ( projENABLE_TRACING == 0 ) */
     static BaseType_t xTraceRunning = pdTRUE;
-#endif
+#endif /* if ( projENABLE_TRACING == 0 ) */
```

Comment:
No sure why we keep the variable `xTraceRunning` in the original design. It should be able to be replaced by `#if ( projeENABLE_TRACING == 1 )`.

Reply:
Yes , it seems unused now with the latest flag for enabling tracing. I will update the PR. Thanks.